src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -637,6 +629,24 @@ public static int CompareOrdinal(String strA, String strB)
             return CompareOrdinalHelper(strA, strB);
         }
 
+        // TODO https://github.com/dotnet/corefx/issues/21395: Expose this publicly?
+        internal static int CompareOrdinal(ReadOnlySpan<char> strA, ReadOnlySpan<char> strB)
+        {
+            // TODO: This needs to be optimized / unrolled.  It can't just use CompareOrdinalHelper(str, str)
```

Comment:
>TODO: This needs to be [](start = 15, length = 22)

do we have issue tracking this one?

Reply:
This is part of https://github.com/dotnet/corefx/issues/21395; exposing the methods entails making them perform as well as possible.